src/main/java/org/broadinstitute/consent/http/db/StudyDAO.java

```diff
@@ -133,6 +133,12 @@ DELETE FROM study WHERE study_id in (select study_id from property_deletes)
       """)
   void deleteStudyByStudyId(@Bind("studyId") Integer studyId);
 
+  @SqlUpdate(
+      """
+      DELETE FROM public.study_property where study_id = :studyId
```

Comment:
We usually don't specify the schema, is there a reason it's needed here? Locally, tests pass without it.

```suggestion
      DELETE FROM study_property where study_id = :studyId
```

Reply:
Good catch.  Not needed.  I think I have the local DB set up in autocomplete and it did it when I pressed tab on the table name.